### PR TITLE
NP Completeness

### LINKED ISSUE #7874: [C_CPP]: NP Completeness
### Description

# NP Completeness Algorithm 

### Domain

C/CPP

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by vishnu95075**
<hr>

## Description 📜

# NP Completeness
- Image refrence example 
- Discription

Issue Fixes #7874

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] C/CPP

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Review by deeqakkk — CHANGES_REQUESTED** [review]
Vishnu, remove all the .png files from the PR and if required embed them in the NP_copleteness.md file itself.

**Comment by vishnu95075** [comment]
@deeqakkk , please review my changes.

**Review by deeqakkk — CHANGES_REQUESTED** [review]
image
fix typo and there is no content under the reduction formula, fix this as well.

File: C_CPP/Algorithmic Approaches/NP Completeness/NP_Completeness.md [hunk 1]
```diff
@@ -47,6 +47,10 @@
 >- **P≠NP** still leave open the average-case-complexity of the problem in **NP.*
 
 ## Reduction formulla 
+![NP-Completeness1](https://user-images.githubusercontent.com/75454756/142078484-b1b0e86d-8c3c-41d9-a6d7-5cf55cf5f169.png)
+
+- [Refrence from geeksforgeeks](https://www.geeksforgeeks.org/np-completeness-set-1/)
```
Comment (deeqakkk) [inline C_CPP/Algorithmic Approaches/NP Completeness/NP_Completeness.md:52]: remove this line as it is already there in reference part.

**Review by deeqakkk — CHANGES_REQUESTED** [review]

**Comment by vasu-1** [comment]
image
Why have you mentioned Pr (#7890) instead of Issue ?

have you made an issue for this Pr ?

**Comment by vishnu95075** [comment]
@vasu-1 **(Issue no: #7874 that is fixes. so that i am mention ther**
gitpng

# Can you see this  is already merged.

**Comment by vasu-1** [comment]
ohk ohk my bad !

**Review by vasu-1 — CHANGES_REQUESTED** [review]
.

**Review by deeqakkk — APPROVED** [review]
@Suvraneel it's good to be merged.
Good work Vishnu and kudos to your patiences🎉👏

**Comment by vishnu95075** [comment]
@vasu-1 , Sorry for that I am image take from **GFG** , I think So it is giving some palagiarism . But I am give refrence for that.

**Pull request merged**